lib/Doctrine/ORM/ORMException.php

```diff
@@ -41,38 +41,6 @@ public static function namedNativeQueryNotFound($nativeQueryName)
         return new self('Could not find a named native query by the name "' . $nativeQueryName . '"');
     }
 
-    /**
-     * @param object $entity
-     * @param object $relatedEntity
-     *
-     * @return ORMException
-     */
-    public static function entityMissingForeignAssignedId($entity, $relatedEntity)
-    {
-        return new self(
-            'Entity of type ' . get_class($entity) . ' has identity through a foreign entity ' . get_class($relatedEntity) . ', ' .
-            'however this entity has no identity itself. You have to call EntityManager#persist() on the related entity ' .
-            "and make sure that an identifier was generated before trying to persist '" . get_class($entity) . "'. In case " .
-            'of Post Insert ID Generation (such as MySQL Auto-Increment) this means you have to call ' .
-            'EntityManager#flush() between both persist operations.'
-        );
-    }
-
-    /**
-     * @param object $entity
-     * @param string $field
-     *
-     * @return ORMException
-     */
-    public static function entityMissingAssignedIdForField($entity, $field)
-    {
-        return new self('Entity of type ' . get_class($entity) . " is missing an assigned ID for field  '" . $field . "'. " .
-            'The identifier generation strategy for this entity requires the ID field to be populated before ' .
-            'EntityManager#persist() is called. If you want automatically generated identifiers instead ' .
-            'you need to adjust the metadata mapping accordingly.');
-    }
-
-    /**
```

Comment:
This introduced Parse error in master. :/ We really need to fix the test suite asap...

Reply:
Fixed ind455a023f8905297f7e8dc9825d1faebb875a48e.

Reply:
Oh wow so sorry!

Reply:
The test suite is red though, so this is on me: https://travis-ci.org/doctrine/doctrine2/builds/332942097?utm_source=github_status&utm_medium=notification